src/main/kotlin/uk/gov/justice/digital/hmpps/visitallocationapi/model/entity/PrisonerDetails.kt

```diff
@@ -34,6 +34,9 @@ open class PrisonerDetails(
   @OneToMany(mappedBy = "prisoner", fetch = FetchType.LAZY, cascade = [CascadeType.PERSIST, CascadeType.MERGE], orphanRemoval = true)
   val changeLogs: MutableList<ChangeLog> = mutableListOf()
 
+  @OneToMany(mappedBy = "prisoner", fetch = FetchType.LAZY, cascade = [CascadeType.PERSIST, CascadeType.MERGE], orphanRemoval = true)
+  val visitOrderHistory: MutableList<VisitOrderHistory> = mutableListOf()
+
```

Comment:
do you think we need this as part of prisoner details? it is lazily loaded but wondering if we need it at all as it will be only used in 1 use case